src/NUnitTestAdapter/NUnit3TestDiscoverer.cs

```diff
@@ -68,19 +68,21 @@ public void DiscoverTests(IEnumerable<string> sources, IDiscoveryContext discove
 
             foreach (string sourceAssembly in sources)
             {
+                var sourceAssemblyPath = Path.IsPathRooted(sourceAssembly) ? sourceAssembly : Path.Combine(Directory.GetCurrentDirectory(), sourceAssembly);
```

Comment:
I was originally using Path.GetFullPath(), but that introduces new error catching concerns, as it requires that the given file actually exists, which hasn't been verified yet. I spotted the "combine with current directory" method in a couple spots already in the code and it seemed like a better way to do it, since it leaves validation to the error-handling code that's already in place.